### ISSUE TITLE
"RC24 migration"

### ISSUE THREAD

**Issue by larsenv** [Difegue/DoujinSoft#44]
My opinions aside... Can you tell me why you're migrating to use WiiLink for mail now? Is there a good reason for it which abandons RiiConnect24's userbase of your site?

**Comment by Difegue** [comment]
I don't believe RC24 is going to receive any further major development considering the differences in dev teams; And looking at the user metrics it wasn't worth the additional development to support both backends at once.

**Comment by larsenv** [comment]
I was quite upset that you removed support honestly. I don't really understand why you'd switch over if there are people which use it on RiiConnect24. For example SuperLeo reached out to me and said he doesn't want to install WiiLink and if you do this then he can't submit records through his Wii. It's true that there probably wouldn't be major developments, but it doesn't make sense to abandon whoever uses it with RiiConnect24.

**Comment by larsenv** [comment]
I was quite upset that you removed support honestly. I don't really understand why you'd switch over if there are people which use it on RiiConnect24. For example Leo reached out to me and said he doesn't want to install WiiLink and if you do this then he can't submit records through his Wii. It's true that there probably wouldn't be major developments, but it doesn't make sense to abandon whoever uses it with RiiConnect24.

**Comment by larsenv** [comment]
I was quite upset that you removed support honestly. I don't really understand why you'd switch over if there are people which use it on RiiConnect24. For example Leo reached out to me and said he doesn't want to install WiiLink and if you do this then he can't submit records through his Wii. It's true that there probably wouldn't be major developments, but it doesn't make sense to abandon whoever uses it with RiiConnect24.

**Comment by larsenv** [comment]
I was quite upset that you removed support honestly. I don't really understand why you'd switch over if there are people which use it on RiiConnect24. For example Leo reached out to me and said he doesn't want to install WiiLink and if you do this then he can't submit records through his Wii. It's true that there probably wouldn't be major developments, but it doesn't make sense to abandon whoever uses it with RiiConnect24.

**Comment by Difegue** [comment]
That's unfortunate - Luckily there's still direct .mio support for now.  
If there's enough demand for it, I might support both backends in parallel in a later version!
